classes/job/casts/DatetimeToInt.php

```diff
@@ -32,6 +32,9 @@ class DatetimeToInt implements CastsAttributes
      */
     public function get($model, $key, $value, $attributes)
     {
+        if(is_numeric($value)) {
+            return Carbon::parse( intval($value) );
+        }
```

Comment:
if we add this check here, we need to add this check in the in the `set` method also .